data/Nishitbaria.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "Nishit Baria",
+  "type": "personal",
+  "displayStatsPublic": true,
+  "bio": "**Open Source** Advocate | Web-Dev | Space Lover ",
+  "tags": ["Open Source", "c++", "Java", "SQL", "Open Source "],
```

Comment:
Looks good. But you have used 'open source' twice in the tag parameter, you may want to update it..

Reply:
Hey Bereky, I solve this problem in short time